catalog/nsili/catalog-nsili-endpoint/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -51,6 +51,27 @@
                 name="Enabled Outgoing Validation" id="outgoingValidationEnabled" required="true" type="Boolean"
                 default="false"
         />
+
+        <AD
+                name="Sources to Query:" id="querySources" description="Configured sources to query from this endpoint. Empty list defaults to local only."
+                required="true" type="String" cardinality="1000"
+                default=""
+        />
+
+        <AD
+                name="Email From Address" id="emailFrom" description="The 'from' address used when sending an email."
+                required="true" type="String" default="donotreply@nowhere.com"
+        />
+
+        <AD     name="Email Subject Line" id="emailSubject" description="The subject line used when sending an email."
```

Comment:
More info here - The only time an email would be sent is if a user requested the product to be delivered via email.